src/js/containers/aboutTheData/AgenciesDetailContainer.jsx

```diff
@@ -16,23 +17,251 @@ import Header from 'containers/shared/HeaderContainer';
 
-const message = 'Data in this table will be updated whenever the underlying data submissions change or new submissions are added.';
+const Tooltip = ({ title }) => (
+    <TooltipComponent title={title}>
+        <p>Place holder for tooltip component.</p>
+    </TooltipComponent>
+);
+
+Tooltip.propTypes = {
+    title: PropTypes.string.isRequired
+};
+
+const columns = [
+    {
+        title: "fiscal_year",
+        displayName: "Reporting Period"
+    },
+    {
+        title: "total",
+        displayName: "Percent of Total Federal Budget",
+        icon: (
+            <TooltipWrapper
+                icon="info"
+                tooltipComponent={<Tooltip title="Percent of Total Federal Budget" />} />
+        )
+    },
+    {
+        title: "publication_date",
+        displayName: "Most Recent Update",
+        icon: (
+            <TooltipWrapper
+                icon="info"
+                tooltipComponent={<Tooltip title="Most Recent Update" />} />
+        )
+    },
+    {
+        title: "missing_tas_count",
+        displayName: "Number of TASs Missing from Account Balance Data",
+        icon: (
+            <TooltipWrapper
+                icon="info"
+                tooltipComponent={<Tooltip title="TASs Missing from Account Balance Data" />} />
+        )
+    },
+    {
+        title: "obligation_different",
+        displayName: "Reporting Difference in Obligations",
+        icon: (
+            <TooltipWrapper
+                icon="info"
+                tooltipComponent={<Tooltip title="Reporting Difference in Obligations" />} />
+        )
+    },
+    {
+        title: "unlinked_cont_award_count",
+        displayName: "Number of Unlinked Contract Awards",
+        icon: (
+            <TooltipWrapper
+                icon="info"
+                tooltipComponent={<Tooltip title="Number of Unlinked Contract Awards" />} />
+        )
+    },
+    {
+        title: "unlinked_asst_award_count",
+        displayName: "Number of Unlinked Assistance Awards",
+        icon: (
+            <TooltipWrapper
+                icon="info"
+                tooltipComponent={<Tooltip title="Number of Unlinked Assistance Awards" />} />
+        )
+    },
+    {
+        title: "assurance_statements",
+        displayName: "Assurance Statements",
+        icon: (
+            <TooltipWrapper
+                icon="info"
+                tooltipComponent={<Tooltip title="Assurance Statements" />} />
+        )
+    }
+];
+
+const mockAPIResponse = {
+    page_metadata: {
+        page: 1,
+        hasNext: false,
+        hasPrevious: false,
+        total: 2
+    },
+    results: [
+        {
+            fiscal_year: "FY 2020: Q2 / P06",
+            percent_total_budget: 41.23,
+            recent_update: "09/29/2020",
+            discrepancy_count: 2,
+            obligation_difference: 0,
+            unlinked_cont_awd: 782,
+            unlinked_asst_awd: 5,
+            assurance_statements: <AgencyDownloadLinkCell />
```

Comment:
The component itself wouldn't be returned from the API, so this line can be removed from each response object